### PR TITLE
Mobile & desktop versions

### PR DESCRIPTION
**Pull request by miltonHenschel**
- [ ] Create home and about pages for mobile version
- [ ] Create home and about pages for desktop version
- [ ] Create and load speakers section dynamically
- [ ] Add transitions and animations to some elements
- [ ] Create mobile menu
- [ ] Add action on more/less button
- [ ] Deploy website

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -22,10 +30,87 @@ function closeMenu() {
...
+          <h4>Professor Emeritus, Korea Advanced Institute of Science and Technology (KAIST)</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>He developed SDN, the first Internet protocol network in Asia, and led Korea's first dedicated line Internet connection in 1990, opening the Internet era in earnest.</p>
+        </div>
+    </div>
+
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_03.png" alt="speaker 3"></div>
+        <div class="speaker-details">
+          <h3>Soyoung Roh</h3>
```
Comment (tresorsawasawa) [inline js/script.js:74]: - [ ] Kindly make sure you implement a function that iterates through the speakers' list on window load and creates HTML elements with the data from each object as required for the project.

- [ ] Also, the dynamic content should be attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's `appendChild()` or `innerHTML()`.
For a short remember about the ` DYNAMIC SPEAKERS`, you can use the same logic as what you did [while creating your portfolio on the **cards' project**](https://github.com/miltonHenschel/myPortfolio/blob/main/js/projects.js).

Comment (tresorsawasawa) [inline js/script.js:74]: - [ ] To match to the template design, kindly make sure you add the missing background grid-mage on each speaker picture.
Screenshot from 2022-11-23 23-54-21

File: js/script.js [hunk 2]
```diff
@@ -22,10 +30,87 @@ function closeMenu() {
...
+          <h3>Ryan Merkley</h3>
+          <h4>CEO of Creative Commons, former COO of the Mozilla Foundation</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>He led open source projects at the Mozilla Foundation and joined CC as CEO in 2014. Open government, open source, and other open movements have been actively carried out.</p>
+        </div>
+    </div>
+  </section>
+
+  <button id="more-speakers">MORE&nbsp;&nbsp;<img src="./images/nav/arrow_down.png" alt="down icon"></button>
+`;
```
Comment (tresorsawasawa) [inline js/script.js:113]: - [ ] Good job adding the `More` button on your mobile design. So, I noticed that on Mobile, only 2 speakers are shown. According to the logic, other speakers should be shown once clicked the `More` button but on your page, it is not the case even if the `MORE` button has been clicked. Make sure you fix that.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @miltonHenschel 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+            <div id="all-rights-detail"><p>All content on the website unless otherwise specified is licensed under a Creative Commons Attribution 2.0 United States License. For details of the functions and conditions of permission http://creativecommons.org/licences/by/2.0/kr</p></div>
+         </div>
+      </section>
+   </footer>
+
+   <script src="js/script.js"></script>
+
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (miltonHenschel) [reply about.html:115]: Yeah, thanks, I'll do just that. I completely forgot.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+         <div id="all-rights-info">
+            <div id="all-rights-intro"><p>2015 Creative Commons Korea. Some Rights Reserved.</p></div>
+
+            <div id="all-rights-detail"><p>All content on the website unless otherwise specified is licensed under a Creative Commons Attribution 2.0 United States License. For details of the functions and conditions of permission http://creativecommons.org/licences/by/2.0/kr</p></div>
+         </div>
+      </section>
+   </footer>
+
+   <script src="js/script.js"></script>
+
```
[the remark this answers is not in the document]
Comment (miltonHenschel) [reply about.html:113]: Yeah, thanks @tresorsawasawa, I'll do just that. I completely forgot.

File: index.html [hunk 5]
```diff
@@ -65,7 +70,7 @@ <h3>Lecture</h3>
             <div class="main-program-div">
                <div class="main-program-img"><img src="./images/main-program/cc-exhibition.png" alt="cc exhibition icon"></div>
                <h3>CC Exhibition</h3>
-               <p>He developed SDN, the first Internet protocol network in Asia, and led Korea's first dedicated line Internet connection in 1990, opening the Internet era in earnest.</p>
+               <p>He developed SDN, the first Internet protocol network in Asia.</p>
             </div>
 
             <div class="main-program-div">
```
[the remark this answers is not in the document]
Comment (miltonHenschel) [reply index.html:76]: Oh, thanks @tresorsawasawa, I'll remove the center-align and left-align them.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+            <ul id="nav-bar-ul">
+               <li class="nav-bar-list"><a href="index.html">Home</a></li>
+               <li class="nav-bar-list"><a href="about.html">About</a></li>
+               <li class="nav-bar-list"><a href="index.html#main-program">Program</a></li>
+               <li class="nav-bar-list"><a href="#">Join</a></li>
+               <li class="nav-bar-list"><a href="#">Sponsor</a></li>
+               <li class="nav-bar-list"><a href="#">CC Campaign</a></li>
+            </ul>
+         </div>
+      </nav>
```
[the remark this answers is not in the document]
Comment (miltonHenschel) [reply about.html:44]: Thanks @tresorsawasawa, but I think it appears like that on your end. Here's how it appears on my screen:

cr1

File: about.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+         <div id="nav-bar-logo"><a href="index.html"><img src="./images/logo/logo.png" alt="cc logo" /></a></div>
+         <div id="nav-bar-div">
+            <ul id="nav-bar-ul">
+               <li class="nav-bar-list"><a href="index.html">Home</a></li>
+               <li class="nav-bar-list"><a href="about.html">About</a></li>
+               <li class="nav-bar-list"><a href="index.html#main-program">Program</a></li>
+               <li class="nav-bar-list"><a href="#">Join</a></li>
+               <li class="nav-bar-list"><a href="#">Sponsor</a></li>
+               <li class="nav-bar-list"><a href="#">CC Campaign</a></li>
+            </ul>
```
[the remark this answers is not in the document]
Comment (miltonHenschel) [reply about.html:42]: Very true! Thanks @tresorsawasawa.

File: about.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+         <h1>"Hello, Sharing World!"</h1>
+
+         <h2>CREATIVE COMMONS GLOBAL SUMMIT 2022</h2>
+
+         <h3>Held every two years, the Creative Commons Global Summit brings together CC activists from around the world and experts, scholars and activists from all walks of life who support and practice open culture.</h3>
+
+         <h4>October 14 ~ 17, 2015 (Wed ~ Sat)<br />See you at the CC Global Summit!</h4>
+
+         <p>Do you have any other questions about CC Global Summit 2015? Please feel free to contact us via email <a href="mailto:ccsummit2015@cckorea.org">ccsummit2015@cckorea.org</a></p>
+      </article>
```
[the remark this answers is not in the document]
Comment (miltonHenschel) [reply about.html:56]: Ok thanks @tresorsawasawa.

File: js/script.js [hunk 9]
```diff
@@ -22,10 +30,87 @@ function closeMenu() {
...
+        <div class="speaker-img"><img src="./images/speakers/speaker_06.png" alt="speaker 6"></div>
+        <div class="speaker-details">
+          <h3>Ryan Merkley</h3>
+          <h4>CEO of Creative Commons, former COO of the Mozilla Foundation</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>He led open source projects at the Mozilla Foundation and joined CC as CEO in 2014. Open government, open source, and other open movements have been actively carried out.</p>
+        </div>
+    </div>
+  </section>
+
```
[the remark this answers is not in the document]
Comment (miltonHenschel) [reply js/script.js:111]: Yes, I will do it @tresorsawasawa.

File: js/script.js (hunk 1 above)
Comment (miltonHenschel) [reply js/script.js:74]: Yes, thanks @tresorsawasawa.

Comment (miltonHenschel) [reply js/script.js:74]: Hmm, tricky but please could you suggest me a way to accomplish @tresorsawasawa? Thanks in advance.

File: js/script.js (hunk 2 above)
Comment (miltonHenschel) [reply js/script.js:113]: Thanks but I think it is an optional requirement @tresorsawasawa. Please check the information below: 

cr2

File: README.md [hunk 10]
```diff
@@ -81,6 +90,7 @@
 
 <!-- > Add a link to your deployed project.-->
 - [Live Demo Link](https://miltonhenschel.github.io/Conference-Page/)
+- [Video presentation](https://www.loom.com/share/ea7190471a844c69bfe0f44099ecec13)
```
Comment (fobadara) [inline README.md:93]: You mentioned that creating the speaker's page dynamically makes the page load faster. On the contrary, it makes the code slower but makes updating your list of speakers faster.

**Review by fobadara — APPROVED** [review]
Hi @miltonHenschel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
✅ Project looks professional
✅ Animations

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 10 above)
Comment (miltonHenschel) [reply README.md:93]: Oh! you watched my video presentation? Thanks for offering your remarks.

Comment (miltonHenschel) [reply README.md:93]: Oh! you watched my video presentation? Thanks for offering your remarks.

Comment (miltonHenschel) [reply README.md:93]: Thanks for the remarks @fobadara.

Comment (miltonHenschel) [reply README.md:93]: Thanks for the remarks @fobadara.

**Pull request merged**